src/App.js

```diff
@@ -11,23 +18,82 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      player1score: 0,
```

Comment:
you should use camelCase for object properties, including states. when you combine numbers with text it can be a bit more tricky to notice where the camel case starts. i would recommend spelling out numbers for names, e.g.

```
playerOneScore
playerTwoScore
```

Reply:
okay changed.